src/components/NewProfileForm.js

```diff
@@ -44,21 +48,25 @@ class NewProfileForm extends Component {
     })
   }
 
+  updateCurrentProfile = () => {
+    const newProfile = {
+      name: this.state.name,
+      email: this.state.email,
+      phone: this.state.business_phone,
+      address: this.state.address,
+      region: this.state.region,
+      bio: this.state.bio,
+      photoUrl: this.state.photo_path
+    }
+    this.setState({ profile: newProfile })
+  }
+
   submitProfile = event => {
     event.preventDefault();
     if (this.state.name && this.state.email && this.state.region && this.state.bio) {
-      const newProfile = {
-        name: this.state.name,
-        email: this.state.email,
-        business_phone: this.state.business_phone,
-        address: this.state.address,
-        region: this.state.region,
-        bio: this.state.bio,
-        photo_path: this.state.photo_path
-      }
-      console.log(newProfile)
+
       // here's where we're gonna write what happens when you submit a profile!
-      this.clearInputs();
+      // this.clearInputs();
```

Comment:
Excellent point! Yesterday, I went through on a new branch and cleaned up all of our files (removed commented out code, streamlined syntax/code styling/indentation, etc...) It will be removed in the feature/cypress-testing branch.